CHANGELOG.md

```diff
@@ -0,0 +1,356 @@
+## Changelog
+
+### Version 1.32
+PLACEHOLDER DATE, 2019
+
+- Migrate changelog from wiki to github, add README ([PR \#113](https://github.com/jenkinsci/durable-task-plugin/pull/113))
+
+### Version 1.31
+Oct 22, 2019
+
+> **WARNING**: This version (1.31) introduced a bug where the binary will not be able to launch the script
+> on non-x86 platforms and container-based agents that do not have access to the agent node's root directory.
+ 
+-   Update ssh-slaves ([PR \#100](https://github.com/jenkinsci/durable-task-plugin/pull/100))
+-   Do not fail tests when run on a machine without Docker installed.
+    ([PR \#101](https://github.com/jenkinsci/durable-task-plugin/pull/101))
+-   Improve watcher logging
+    ([PR \#102](https://github.com/jenkinsci/durable-task-plugin/pull/102))
+-   Refactor UNIX unit tests for greater test coverage
+    ([PR \#103](https://github.com/jenkinsci/durable-task-plugin/pull/103))
+-   Allow setting pwsh as Powershell executable
+    ([PR \#112](https://github.com/jenkinsci/durable-task-plugin/pull/111))
+-   Bugfix: Use setsid instead of nohup ([JENKINS-25503](https://issues.jenkins-ci.org/browse/JENKINS-25503))
```

Comment:
Important to mention the system property that allows this behavior to be disabled.